osu.Game/Tests/Visual/OsuGameTestScene.cs

```diff
@@ -169,7 +176,7 @@ public TestOsuGame(Storage storage, IAPIProvider api, string[] args = null)
                 : base(args)
             {
                 base.Storage = storage;
-                API = api;
+                API = new DelegatedAPI(api);
```

Comment:
This change is quite important. In the case of nested games, we would be binding bindables in those nested games to the global `TestScene`'s API which will persist through multiple tests.

This `DelegatedAPI` isolates those bindables.

Reply:
probably best to force push the old fix out of history

Reply:
Fair call, done.